### PR TITLE
[src] Make ivector-extract-online2.cc ignore pitch in case of dim mismatch

### PR DESCRIPTION
**Pull request by danpovey**

### CODE REVIEW

**Comment by danpovey** [comment]
Tested, see  https://groups.google.com/d/msgid/kaldi-help/87d3d713-db02-4caf-823e-b78f2575b045%40googlegroups.com

File: src/online2/online-ivector-feature.cc [hunk 1]
```diff
@@ -67,6 +67,15 @@ void OnlineIvectorExtractionInfo::Init(
   this->Check();
 }
 
+int32 OnlineIvectorExtractionInfo::ExpectedFeatureDim() const {
+  int32 num_splice = 1 + splice_opts.left_context + splice_opts.right_context,
+      full_dim = lda_mat.NumCols();
+  if (!(full_dim % num_splice == 0) || (full_dim % num_splice == 1)) {
```
Comment (francisr) [inline src/online2/online-ivector-feature.cc]: Should be: `if (!(full_dim % num_splice == 0 || full_dim % num_splice == 1)) {`

Comment (danpovey) [reply src/online2/online-ivector-feature.cc]: Thanks !!! Fixing

**Comment by francisr** [comment]
Is there anything that prevents merging this?

**Comment by danpovey** [comment]
sorry, I thought I had already merged it.  Merging now.

**Pull request merged**